src/App.tsx

```diff
@@ -1,24 +1,71 @@
 import React from 'react';
 import './App.scss';
 
-const App: React.FC = () => {
-  setInterval(() => {
+type Props = {
+
+};
+
+type State = {
+  isClockVisible: boolean,
+  clock: string,
+};
+
+class App extends React.Component<Props, State> {
+  state = {
+    isClockVisible: true,
+    clock: '',
+  };
+
+  componentDidMount() {
+    setInterval(() => {
+      if (this.state.isClockVisible) {
+        this.Clock();
+      } else {
+        this.hideClock();
+      }
+    }, 1000);
+  }
+
+  hideClock = () => {
+    this.state.isClockVisible = false;
+    this.setState({ clock: ' ' });
+  };
+
+  showClock = () => {
+    this.state.isClockVisible = true;
+  };
+
+  Clock = () => {
     const date = new Date();
 
     // eslint-disable-next-line
     console.log(date.toLocaleTimeString());
```

Comment:
```suggestion
    const date = new Date();

    // eslint-disable-next-line
    console.log(date.toLocaleTimeString());
```

These operation should be inside the setInterval method